docs/config.md

```diff
@@ -1229,8 +1229,22 @@ The `manifest` scope allows you to define some meta-data information needed when
 The following settings are available:
 
 `manifest.author`
+: :::{deprecated} 24.09.0-edge
+  Use `manifest.contributors` instead.
+  :::
 : Project author name (use a comma to separate multiple names).
 
+`manifest.contributors`
+: :::{versionadded} 24.09.0-edge
+  :::
+: List of project contributors. Should be a list of maps. The following fields are supported in the contributor map:
+  - `name`: the contributor's name 
+  - `affiliation`: the contributor's affiliated organization
+  - `email`: the contributor's email address
+  - `github`: the contributor's GitHub URL
+  - `contribution`: list of contributions, can be any of `'author'`, `'maintainer'`
```

Comment:
List of contributions? Or type of contribution? It has to be a string, so people have to choose right?

Reply:
Ah no it can be multiple, nice 👍🏻 
```suggestion
  - `contribution`: list of contribution types, can include any of `'author'`, `'maintainer'`
```

Reply:
ok, so it can be `'author'`, `'maintainer'`, or `['author', 'maintainer']`.

Reply:
Yeah, always a list though. So `['author']`

Reply:
Gotcha

Reply:
`List of contribution types`, is much more understandable than `List of contributions`, thanks for the suggestion

Reply:
Or `contribution roles`? 

As actually I personally would have three roles: `author` (original person who started), `maintainer` (someone who is currently leading the development) and a `contributor` (someone who occasionally/regularly throws things in)

Reply:
I like @jfy133 's thinking, though maybe contributor is harder to map to Ben's ontology.

Reply:
I'm not super opinionated about the contribution types, as long as it doesn't cause problems with supporting things like BCO

"author" and "maintainer" seem to map to "authoredBy" and "curatedBy". if we add "contributor", I could just map that also to "curatedBy" in the BCO. that's not the end of the world, but it would minimize the contribution of "maintainers"

also, I wonder if it's necessary to include small contributors in the manifest when you could just credit them in the README

Reply:
By occasionally/regularly I didn't necessarily mean small, but rather a person who is not 'running' or 'leading' the point of view.

For example, I currently do most of the day to do managing of bug reports and fixes, user support, and project planning, in nf-core/mag (maintainer), but then I often have @prototaxites helping with reviews and every month or so dropping a chunky bit of new functionality (contributor). 

To map to an analogy from academic publishing: the maintainer is the first author (at the given time point), but then a contributor is someone who has still done a good chunk of work worthy of co-authorship in the middle of the list. Versus someone who made a couple of small fixes or docs improvements who would not go into acknowledgements (which here would go into the README).

I think that still maps to the ontology, what I'm proposing is more of a human interpretation thing to make sure the level of acknowledgement of peoples input is fair.

Reply:
in that case I would be fine with adding "contributor" and mapping it also to "curatedBy" in the BCO. the kind of contributions you mentioned are comparable to that of a maintainer IMO